code/modules/modular_armor/helmets.dm

```diff
@@ -99,10 +99,12 @@
 
 /obj/item/helmet_module/binoculars/onzoom(mob/living/user)
 	RegisterSignal(user, list(COMSIG_MOVABLE_MOVED, COMSIG_MOB_MOUSEDOWN), .proc/toggle_module)//No shooting while zoomed
+	RegisterSignal(user, COMSIG_ITEM_ZOOM, .proc/zoom_check_return)
```

Comment:
i dont like this not being parent called or so since someone will inevitably forget to add it

Reply:
i could probably add this to the zoom proc itself

Reply:
still not fixed